cli/rt/02_console.js

```diff
@@ -481,6 +481,19 @@
     return quoteString(string);
   }
 
+  // Surround a symbol's description in quotes when it is required (e.g the description has non printable characters).
+  function maybeQuoteSymbol(symbol) {
+    if (symbol.description === undefined) {
+      return symbol.toString();
+    }
+
+    if (/^[a-zA-Z_][a-zA-Z_.0-9]*$/.test(symbol.description)) {
```

Comment:
```suggestion
    if (/^[$a-zA-Z_][$a-zA-Z_.0-9]*$/.test(symbol.description)) {
```

Reply:
Fine with changing it but just want to point out that I deliberately omitted this from here and from key strings to match Node's behaviour (in respect to key strings, Symbol printing is wacky across the board).